src/applications/vaos/services/location/transformers.js

```diff
@@ -19,6 +19,7 @@ export function transformDSFacilities(facilities) {
   return facilities.map(facility => ({
     resourceType: 'Location',
     id: facility.id,
+    vistaId: facility.rootStationCode,
```

Comment:
Same name as in our Appointment object and in the /facilities endpoint in mobile facility service